docs/smart-agent-quickstart.md

```diff
@@ -1,132 +1,298 @@
-# Smart Agent Quick Start
+# SignalFx Smart Agent Next Steps
 
-- [Deploy Directly on Host](#deploy-directly-on-host)
 
+__See [Overview](https://docs.signalfx.com/en/latest/integrations/agent/overview.html) for Smart Agent Installation on a Single Host.__
 
-## Deploy Directly on Host
 
-This tutorial assumes you are starting fresh and have no existing collectd agent running on your instance.
+After you have installed the SignalFx Smart Agent on a single host and discovered some of its capabilities, you may want to install the agent on multiple hosts. Those next steps are discussed below.
 
-Certain installation statements include `YOUR_SIGNALFX_REALM`. If this value is not set, SignalFx assumes your organization is in the us0 realm. To determine if you are in a different realm and need to supply your realm value in those statements, check your profile page in the SignalFx web application. 
+## Step 4. Install Smart Agent on Multiple Hosts
```

Comment:
The info in this file isn't really "steps" per se, as they are option and un-ordered.

Reply:
This is actually the order that Joe came up with to recommend, as they are logical for a new user to expand his/her knowledge.

Reply:
That's fine, but I still don't think they need to be numbered. Plus it's weird to start a doc at Step 4.

Reply:
Ok, I deleted the steps on this page. Added some white space to set off each new subject.